dstep/translator/MacroDefinition.d

```diff
@@ -0,0 +1,1429 @@
+/**
+ * Copyright: Copyright (c) 2016 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Jun 03, 2016
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+
+module dstep.translator.MacroDefinition;
+
+import std.array : Appender;
+
+import clang.c.Index;
+import clang.Cursor;
+import clang.Token;
+
+import dstep.translator.Context;
+import dstep.translator.Output;
+
+/**
+  * The accept family of functions parse tokens. In the case of successful parsing
```

Comment:
Please align the `*` to the first `*` in `/**`. This will match the module header/documentation.